spotify_player/src/state/ui/popup.rs

```diff
@@ -37,8 +37,8 @@ pub enum ActionListItem {
 /// An action on an item in a playlist popup list
 #[derive(Debug)]
 pub enum PlaylistPopupAction {
-    Browse,
-    AddTrack(TrackId<'static>),
+    Browse(i32),
+    AddTrack(i32, TrackId<'static>),
```

Comment:
I assume this represents folder level, which cannot be negative. Should be represented as `usize` instead

Reply:
Done